modules/data-protection/backup-vault/main.bicep

```diff
@@ -54,6 +54,12 @@ param azureMonitorAlertSettingsAlertsForAllJobFailures string = 'Enabled'
 @description('Optional. List of all backup policies.')
 param backupPolicies array = []
 
+@description('Optional. Security settings for the backup vault.')
+param securitySettings object = {}
+
+@description('Optional. Feature settings for the backup vault.')
+param featureSettings object = {}
```

Comment:
I know the doco is outdated, but if we are introducing two new params, is there something we can add them to the test file at least so it can be visible the readme on how to use them?

Reply:
Hey @ahmadabdalla , I wish, that was plan A. But I didn't find a single example (not even in the REST documention, which is usually a great reference).